### ISSUE TITLE
[Bug]: About module accessibility issues

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ods-ios#597]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Plus

### OS Version

17.1.1

### App version

v0.16.0

### Describe the issue

This list is about "About" module. Other common issues are described in dedicated issues.

- [x] #678
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/d7076f84-774b-4b8c-bc53-a7e5086d7512)
**Recommandation: Check Dynamic Type activation on components**

- [x] #763
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/98037738-f55a-456a-af69-81750b0c190e)
**Recommandation: Add heading trait to texts which look like a header**

- [x] #677
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/e92c486b-b44a-4eb0-87ac-8be287669bc5)
**Recommandation: Group label and buttons to ease the vocalization**

- [x] Selected/not-selected status of options is not vocalized
**Recommandation: Add accessible value for selected/not-selected buttons**

- [x] Keyboard focus not visible on selected buttons 
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/418c38c0-81bc-46d3-a600-74bdf017fd36)
**Recommandation: Add an outline to make the focus visible on selected buttons**

### Expected Behavior
Page 100% accessible
